src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,182 @@
+package storage;
+
+import Dictionary.Word;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Stack;
+import java.util.TreeMap;
+
+/**
+ * Represents the object that reads and writes to the text files where data is stored.
+ */
+public class Storage {
+
+    private static String FILE_PATH;
+
+    public Storage() {
+    }
+
+    public Storage(String filePath) {
+        FILE_PATH = filePath;
+    }
+
+    /**
+     * Convert all data from text file in storage to list of words
+     * @return an arraylist containing all words in dictionary
+     */
+    public TreeMap<String, Word> loadFile() {
+        File file = new File(FILE_PATH);
+        FileReader fr = null;
+        BufferedReader br = null;
+        try {
+            fr = new FileReader(file);
+            br = new BufferedReader(fr);
+            TreeMap<String, Word> wordBank = new TreeMap<>();
+            String line = br.readLine();
+            while (line != null) {
+                // get data from storage
+                // parse the line first
+                if (line.equals("")) {
+                    line = br.readLine(); continue;
+                }
+                String[] parsedWordAndMeaning = line.split(":");
+                Word word = new Word(parsedWordAndMeaning[0].trim(), parsedWordAndMeaning[1].trim());
+                wordBank.put(word.getWord(), word);
+                line = br.readLine();
+            }
+            return wordBank;
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        } finally {
+            try {
+                br.close();
+                fr.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public Stack<Word> loadHistoryFromFile() {
```

Comment:
Header comment. Make explicit the difference between loadHistory and loadWordBank (Stack structure, why)
Also, the code is very similar. Can both be done in one method, or are there times you have to load each separately?